.github/workflows/tests.yml

```diff
@@ -0,0 +1,18 @@
+name: "Running Tests"
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: cachix/install-nix-action@v9
+    - uses: cachix/cachix-action@v6
+      with:
+        name: digitallyinduced
+        signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
+        authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
+    - run: |
+          nix-shell ./NixSupport/shell.nix --run "runghc $(make -f lib/IHP/Makefile.dist print-ghc-extensions) ./Test/IDE/SchemaDesigner/ParserSpec.hs"
```

Comment:
This is missing `Test/ValidationSupport/ValidateFieldSpec.hs`. Maybe use something like `find Test -name '*Spec.hs'` for auto-discovery?

Reply:
Good point. I was thinking we add a `Test.sh` later on where we can add all tests but even though this is a great solution now :) Added it in https://github.com/digitallyinduced/ihp/pull/84/commits/256a8da7c125d2d499c3d32a7ee6d9d94a2b3156 :)